src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Excavator excavator = new Excavator();
```

Comment:
be more attentive, we need to use an abstract reference instead of a specific one 
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop